candidate_profile/forms.py

```diff
@@ -22,7 +23,19 @@ class Meta:
             "veteran",
         )
 
-        help_texts = {"resume": "Allowed file types: .pdf, .doc, .docx"}
+        widgets = {
+            "resume": file_resubmit.widgets.ResubmitFileWidget(
+                attrs={
+                    "accept": "application/pdf, application/msword, \
+                    application/vnd.openxmlformats-officedocument.\
+                    wordprocessingml.document"
+                }
+            )
+        }
+        help_texts = {
+            "resume": "Allowed file types: .pdf, .doc, .docx <br/> "
+            "Max file size: 2 MiB"
```

Comment:
@ahanif16 Do you mean MiB or MB? MiB is apparently Mebibytes (MiB), which I just googled (https://tech.slashdot.org/story/01/12/23/1421225/megabytes-mb-or-mebibytes-mib). I'm more familiar with Megabytes (MB)

Reply:
I had originally written MB here, but it should be MiB given we're dividing by 1024. Can switch this and the size limit to MB if we prefer to do that

Reply:
it's probably better to go with MB because of the familiarity to the user

Reply:
Sounds good. Will make that update